lambda/sendIccidToQueue.ts

```diff
@@ -0,0 +1,15 @@
+import type { APIGatewayProxyResultV2 } from 'aws-lambda'
+import { toStatusCode } from './queueJob.js'
+import { res } from '../api/res.js'
+import { q } from './getBasicInformationLambda.js'
+
+export const sendIccidToQueue = async (
```

Comment:
Result is never used in calling functions.

Reply:
This whole file seems a bit pointless.  Not all callers use the response, so get rid of it and use `q` directly in the lamba.